src/main/java/cms/gov/madie/measure/utils/TestCaseServiceUtil.java

```diff
@@ -309,7 +309,10 @@ && isEmpty(group.getStratificationValues()))
 
     List<TestCaseStratificationValue> stratification =
         testCaseGroupPopulations.stream()
-            .filter(group -> isNotEmpty(group.getStratificationValues()))
+            .filter(
+                group -> {
+                  return isNotEmpty(group.getStratificationValues());
+                })
```

Comment:
Is there a boon to wrapping the lambda expression?

Reply:
Not really.. I just had it in a block so I could add some addition steps when I was debuggin.. I'll remove it